src/httomo/task_runner.py

```diff
@@ -148,8 +148,8 @@ def run_tasks(
             # Check for any extra params unrelated to tomopy but related to
             # HTTomo that should be added in
             httomo_params = \
-                _check_signature_for_httomo_params(func, possible_extra_params)
-
+                _check_signature_for_httomo_params(func, possible_extra_params)            
```

Comment:
Looks like some random extra whitespace at the end of this line got added in? :stuck_out_tongue_closed_eyes: